src/commands/dev-env-sync-sql.ts

```diff
@@ -137,29 +142,29 @@ export class DevEnvSyncSQLCommand {
 		} );
 	}
 
-	generateSearchReplaceMap() {
+	private generateSearchReplaceMap(): void {
 		this.searchReplaceMap = {};
 
 		for ( const url of this.siteUrls ) {
 			this.searchReplaceMap[ url ] = this.landoDomain;
 		}
 
-		const networkSites = this.env.wpSitesSDS.nodes;
+		const networkSites = this.env.wpSitesSDS?.nodes;
 		if ( ! networkSites ) return;
 
 		for ( const site of networkSites ) {
-			if ( ! site.blogId || site.blogId === 1 ) continue;
+			if ( ! site?.blogId || site.blogId === 1 ) continue;
 
-			const url = site.homeUrl.replace( /https?:\/\//, '' );
-			if ( ! this.searchReplaceMap[ url ] ) continue;
+			const url = site?.homeUrl?.replace( /https?:\/\//, '' );
+			if ( ! url || ! this.searchReplaceMap[ url ] ) continue;
```

Comment:
Here's a logic change, and it should be pretty safe, as `this.searchReplaceMap[ url ]` where `url` is falsey is most likely a falsey as well.